Sources/System/Internals/Constants.swift

```diff
@@ -13,8 +13,11 @@
 
 #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
 import Darwin
-#elseif os(Linux) || os(FreeBSD) || os(OpenBSD) || os(Android)
+#elseif canImport(Glibc)
 import Glibc
+#elseif canImport(Musl)
+import CSystem
```

Comment:
in other spots it is imported with @_implementationOnly, also, do we need this in the Glibc case as well?

Reply:
Not sure about this one, I'd stick to how other imports are done in this file for consistency.